docs/reference/supported-environments.md

```diff
@@ -156,3 +156,19 @@ You can also use newer versions of Desktop and Web Modeler with older versions o
 </TabItem>
 
 </Tabs>
+
+## Dependency maintenance policies
+
+Camunda provides [a standard support policy](https://camunda.com/release-policy/) of 18 months for a particular minor version, from the date it is released.
+During this time, patches are regularly released which contain security and bug fixes, some of which may come from dependency updates. As such, for the
+vast majority of dependencies, Camunda _only_ applies patch updates.
+
+Certain dependencies used by Camunda 8, however, may have a shorter maintenance policy than Camunda itself. For those dependencies which are integral to its functionality, Camunda may adopt a different update policy, as listed below.
+
+### Spring
+
+Many server side components, such as the **Camunda Orchestration Cluster**, are Spring Boot applications, and leverage Spring Boot (and the wider Spring ecosystem) to execute fundamental functionality such as application configuration, REST infrastructure (including security), production ready features, etc.
```

Comment:
This is a good place to link to the Camunda Orchestration Cluster, as defined in the glossary. We should likely make it very clear how we have defined this for both current and new users, but I just realized we haven't added this to the glossary yet.

I would propose we provide a link to this glossary definition in every instance of Camunda Orchestration Cluster for now -- additionally, is this the casing we want to standardize on? CC @MaxTru @akeller 

@npepinpe This is out of scope for this PR, so you can disregard.

Reply:
We are still working on some of the terminology in 8.8, but I'm pretty confident this will just be "Orchestration cluster" with no "Camunda" qualifier on the front. I definitely think it's a good opportunity in 8.8 with the intro of the Orchestration cluster that we link to the glossary definition.

Reply:
Related - https://github.com/camunda/camunda-docs/pull/5537